src/server.js

```diff
@@ -211,6 +211,45 @@ app.get(
   }
 );
 
+function averageGeolocation(coords) {
+  if (coords.length === 0) {
+    return {};
+  }
+  if (coords.length === 1) {
+    return {
+      latitude: coords[0].lat,
+      longitude: coords[0].long,
+    };
+  } else {
```

Comment:
as per my previous comment, this else is unnecessary and mixing styles. prefer:

if (0) {
  return ...
}
if (1) {
  return ...
}

return ...


Note that this removes an indentation level as well, which is valuable when trying to avoid line wraps.